src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml

```diff
@@ -2,6 +2,10 @@ dashboardhub_app_homepage:
     path:     /
     defaults: { _controller: DashboardHubAppBundle:Default:index }
 
+dashboardhub_app_dashboard:
+    path:     /d/{uid}
```

Comment:
ok, so what do you want to do

if you want to be gansta and with the kids leave it as `d` :P

On Fri, Mar 13, 2015 at 8:58 AM, Eddie Abou-Jaoude <notifications@github.com
> wrote:

> In src/DashboardHub/Bundle/AppBundle/Resources/config/routing.yml
> <https://github.com/DashboardHub/PipelineDashboard/pull/75#discussion_r26371744>
> :
>
> > @@ -2,6 +2,10 @@ dashboardhub_app_homepage:
> >      path:     /
> >      defaults: { _controller: DashboardHubAppBundle:Default:index }
> >
> > +dashboardhub_app_dashboard:
> > +    path:     /d/{uid}
>
> True. I did try that, but always had some issue. So parked it all on one
> route for now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DashboardHub/PipelineDashboard/pull/75/files#r26371744>
> .
>